python/ray/data/impl/shuffle.py

```diff
@@ -28,6 +29,9 @@ def simple_shuffle(input_blocks: BlockList,
         map_ray_remote_args = {}
     if reduce_ray_remote_args is None:
         reduce_ray_remote_args = {}
+    if "scheduling_strategy" not in reduce_ray_remote_args:
+        reduce_ray_remote_args[
+            "scheduling_strategy"] = SPREAD_SCHEDULING_STRATEGY
```

Comment:
Use the "SPREAD" syntax?

Reply:
SPREAD_SCHEDULING_STRATEGY is a variable for "SPREAD". Do you mean use "SPREAD" literal directly?

Reply:
Yeah, that's what we document in the API, so we should prefer using that. But not a strong preference